sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -121,13 +131,15 @@ def _transmit(self, envelopes: List[TelemetryItem]) -> ExportResult:
         if len(envelopes) > 0:
             try:
                 track_response = self.client.track(envelopes)
-                if not track_response.errors:
+                if not track_response.errors:  # 200
                     self._consecutive_redirects = 0
+                    if self._check_stats_collection():
+                        _update_requests_map('success')
```

Comment:
Since it seems these are standardized, I think we should set up an enum or at least a constant.

Reply:
Changed this to use `_REQ_SUCCESS_NAME` constant.

Reply:
Perfect